cmdstanpy/model.py

```diff
@@ -85,14 +105,32 @@ def __init__(
                         ' found: {}'.format(self._name, exename)
                     )
             self._exe_file = exe_file
-        if platform.system() == "Windows":
+
+        if include_paths is not None:
+            bad_paths = [d for d in include_paths if not os.path.exists(d)]
+            if any(bad_paths):
```

Comment:
Noticed the same `bad_paths` check in the compile method below lines 222-233. Is the second check intended or should this be extracted into a helper function?